clang/include/clang/Basic/AttrDocs.td

```diff
@@ -7483,7 +7483,6 @@ generation of the other destruction cases, optimizing the above `foo.destroy` to
   }];
 }
 
-
```

Comment:
nit: unnecessary changes.

Reply:
Acknowledged. Keeping it to match the 1 line space style of the file.